inspirehep/modules/migrator/legacy/remoteaccount.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""RemoteAccount dump functions."""
+
+from __future__ import absolute_import, print_function
+
+from sqlalchemy.orm.exc import NoResultFound, MultipleResultsFound
+
+
+def get(*args, **kwargs):
+    """Get remoteaccounts."""
+    from invenio_accounts.models import User
+    q = User.query
+    return q.count(), q.all()
+
+
+def dump(u, from_date, **kwargs):
+    """Dump the remoteaccounts as a list of dictionaries."""
+    from invenio_oauthclient.models import RemoteAccount
+    from invenio_accounts.models import UserEXT
+
+    # First fetch all the remote account information
+    remoteaccount = {}
+    try:
+        result = RemoteAccount.query.filter_by(user_id=u.id).one()
```

Comment:
Thing is that we are updating the scope of the ORCID authentication so we would like users to relogin and get a new token once we deploy Invenio 3.